arch/arm/core/aarch64/macro.inc

```diff
@@ -8,29 +8,6 @@
 #define _MACRO_H_
 
```

Comment:
If we add a public version of `macro.inc` under `include/arch/arm/aarch64`, we can do `#include <arch/arm/aarch64/macro.inc>` here.

Reply:
May be I can add the  `.inc` header to the  `arch.h`  so that including `<arch/cpu.h>` should do the job.

Reply:
Sure, that would work too. If we are ever only going to have one "very generic" macro header, that would be more desirable, actually. No strong opinion on this.